### PR TITLE
Feature/bc 424 soft launch landing page skeleton

### PR DESCRIPTION
**Pull request by jasonmar08**
- Set up placeholder content on landing page
- Included SignUp component and added some styling conditional for navbar

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import Logo from '../../assets/Logo.svg'
```
Comment (yeezick) [inline src/components/Footer/Footer.tsx:3]: use shortened paths if you're importing to/from files within the `src` directory
`assets/Logo.svg`

File: src/layout/Layout.tsx [hunk 2]
```diff
@@ -1,14 +1,16 @@
 import React, { useEffect, useState } from 'react'
-import { Loader } from 'components/Loader/Loader'
-import { verify } from 'utilities/api'
-import { useAppDispatch, useAppSelector } from 'utilities/redux/hooks'
+import { Loader } from '../components/Loader/Loader'
```
Comment (yeezick) [inline src/layout/Layout.tsx:2]: same here for paths

File: src/layout/Nav/Nav.scss [hunk 3]
```diff
@@ -1,3 +1,21 @@
+.navbar-colored {
+  background-color: #353131e8;
```
Comment (yeezick) [inline src/layout/Nav/Nav.scss:2]: alphabetize pls
i can imagine a script that would enable us to do this automatically with the pre-commit feature

File: src/layout/Nav/Nav.tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { useState, useEffect } from 'react'
-import { Link } from 'react-router-dom'
+import { Link, useLocation } from 'react-router-dom'
 // TODO: see if path can be shortened
```
Comment (yeezick) [inline src/layout/Nav/Nav.tsx:3]: we shortened the paths so this comment can go >:^)

**Review by yeezick — CHANGES_REQUESTED** [review]
only semantic/linting requests from me, great work!

**Comment by jasonmar08** [comment]
Addressed comments and pushed up with updates. Ready for review/approval!

**Review by yeezick — APPROVED** [review]

**Pull request merged**